app/views/examples/pass-data.html

```diff
@@ -0,0 +1,68 @@
+{% extends "layout.html" %}
+
+{% block page_title %}
+  Example - Passing data
+{% endblock %}
+
+{% block content %}
+<main id="content" role="main">
+
+  {% include "includes/breadcrumb_examples.html" %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <h1 class="heading-xlarge">
+        Passing data
+      </h1>
+
+      <p>
+        You may want to use or display data a user has entered over a few screens. The kit automatically stores all data entered, so you can show it later.
```

Comment:
I think you could have more detail here, or in a later paragraph. 'The kit automatically stores all data entered in to form fields, using the id / name' etc

You could then include a code snippet showing this with the named field.